docs/application/design/introduction/styles/animations.md

```diff
@@ -22,15 +22,15 @@ The concept of fluid animation design represents the look and feel of a gentle b
 
 Animations on the screen should have a consistent direction. Disorienting animations create visual noise and confuse users. Uniform animation effects that take place in a timely manner can maximize usability by enhancing understanding of what is occurring on the screen.
 
-> Avoid overusing animations. Add animations to tasks only if they are related to the task and can enhance users’ understanding of the task. Meaningless, irrelevant animations only add confusion.
+> Avoid overusing animations. Add animations to tasks only if they are related to the task and can enhance users' understanding of the task. Meaningless, irrelevant animations only add confusion.
```

Comment:
It looks like `users'` should changed into `user's`.

Reply:
Please ignore this review : https://english.stackexchange.com/questions/36657/user-s-guide-vs-users-guide.